app/scripts/modules/core/src/task/tasks.html

```diff
@@ -43,7 +40,7 @@ <h3>Task History</h3>
           <span class="glyphicon glyphicon-remove" uib-tooltip="clear search filter"></span>
         </a>
       </div>
-      <div class="col-md-3 text-right" style="padding-left: 0">
+      <div class="col-md-3 text-right" style="padding-left: 0;margin-left: auto">
```

Comment:
worth migrating this layout over to flexbox?  Also, I noticed this problem when text is entered

<img width="1057" alt="Screen Shot 2020-06-22 at 12 36 35 PM" src="https://user-images.githubusercontent.com/2053478/85328237-3faa5900-b485-11ea-9517-5b83cf92c629.png">


old behavior:

<img width="1415" alt="Screen Shot 2020-06-22 at 12 38 52 PM" src="https://user-images.githubusercontent.com/2053478/85328291-5c469100-b485-11ea-9285-4012dacba3ea.png">

Reply:
I have those changes coming in my next push :)